src/zip.h

```diff
@@ -331,7 +331,28 @@ extern int zip_extract_stream(const char *stream, size_t size, const char *dir,
  *
  * @return the zip archive handler or NULL on error
  */
-extern struct zip_t *zip_open_stream(const char *stream, size_t size);
+extern struct zip_t *zip_open_stream(const char *stream, size_t size,
+                                     int level, char mode);
+
+/**
+ * Copy zip archive stream output buffer.
+ *
+ * @param zip zip archive handler.
+ * @param buf output buffer. User should free buf.
+ * @param bufsize output buffer size (in bytes).
+ *
+ * @return copy size
+ */
+extern ssize_t zip_stream_cpy(struct zip_t *zip, void **buf, ssize_t *bufsize);
```

Comment:
For consistency, please rename it to `zip_copy_stream`

Reply:
OK